frontend/src/Order.tsx

```diff
@@ -99,8 +99,19 @@ const Order = () => {
         setIsSubmitPopupOpen(false)
     }
 
+    const setFormToInitialState = () => {
```

Comment:
Maybe we should connect this initial state to the actual initialization of the form so that if we want to change it we only change it in one place as opposed to having to change this method AND the default assignments in the state declarations?

Reply:
Replaced states for each field in the form with a single state for the entire form. Consequently it is possible to set the state of the form to the object that is used to initialize the form.